src/PatternLab/PatternData/Helpers/PatternCodeHelper.php

```diff
@@ -82,6 +82,7 @@ public function run() {
 				$exportClean                      = (isset($options["exportClean"])) ? $options["exportClean"] : false;
 				$data["patternLabHead"]           = (!$this->exportFiles) ? $stringLoader->render(array("string" => $htmlHead, "data" => array("cacheBuster" => $data["cacheBuster"]))) : "";
 				$data["patternLabFoot"]           = (!$this->exportFiles) ? $stringLoader->render(array("string" => $htmlFoot, "data" => array("cacheBuster" => $data["cacheBuster"], "isPattern" => true, "patternData" => json_encode($patternData)))) : "";
+				$data['patternPartial']           = $patternData['patternPartial'];
```

Comment:
Should probably have a check in case the $data is set on the .json level. unless there are reserved data keys?